openy.install

```diff
@@ -1002,3 +1002,24 @@ function openy_update_8068() {
       ->save(TRUE);
   }
 }
+
+/**
+ * Remove blazy for icons.
+ */
+function openy_update_8069() {
```

Comment:
This update should be in **openy_media_image**

Reply:
@miroslav-lee I see 1 commit that not related to this issue - **[Fix bug query of Popups form](https://github.com/ymcatwincities/openy/pull/1408/commits)**. Please remove it from PR and squash all changes to 1 commit.

Reply:
I'm done